runtime/doc/deprecated.txt

```diff
@@ -18,6 +18,44 @@ API ~
 *nvim_command_output()*		Use |nvim_exec()| instead.
 *nvim_execute_lua()*		Use |nvim_exec_lua()| instead.
 
+LSP Diagnostics ~
```

Comment:
the sections here are alphanumberically ordered, can you move this just before the "Lua" section